enterprise/utils.py

```diff
@@ -438,7 +408,43 @@ def get_notification_subject_line(course_name, template_configuration=None):
         return stock_subject_template.format(course_name=course_name)
 
 
-def send_email_notification_message(user, enrolled_in, enterprise_customer, email_connection=None):
+def find_enroll_email_template(enterprise_customer, template_type):
+    """
+    Find email template from the template database represented by EnrollmentNotificationEmailTemplate model.
```

Comment:
Can you add a docstring note for what 'template_type' is, and should it have a default kwarg?

Reply:
added docstring! I decided it's best to make the args required so in fact added raise's to check inputs. Since for this function passing in both is the only correct way to use it.